docs/src/main/sphinx/release/release-427.md

```diff
@@ -0,0 +1,110 @@
+# Release 427 (25 Sep 2023)
+
+## General
+
+* Add support for IPv4 addresses in comparisons using the `contains` functions. ({issue}`18497`)
+* Make plans with complex aggregations deterministic. ({issue}`19103`)
+* Improve performance of hashing operations used by `GROUP BY`, `DISTINCT`, and
+  other aggregations. ({issue}`19059`)
+* Fix query failure due to exceeding node memory limits with `INSERT`
+  statements. ({issue}`18771`)
+* Fix incorrectly reporting nanoseconds instead of milliseconds in query plans. ({issue}`19047`)
+* Fix query hang for certain `LIKE` patterns involving a mix of `%` and `_`. ({issue}`19146`)
+
+## Security
+
+* Add access control check to redirected tables when accessing table comments. ({issue}`18514`)
+
+## Delta Lake connector
+
+* Add support for reading tables with deletion vectors. ({issue}`16903`)
+* Add support for Delta Lake writer
+  [version 7](https://docs.delta.io/latest/versioning.html#features-by-protocol-version). ({issue}`15873`)
+* Add support for writing columns with the `timestamp` type. ({issue}`16927`)
+* Reduce data read from Parquet files for queries with filters. ({issue}`19032`)
+* Improve performance of writing to Parquet files. ({issue}`19122`)
+* Fix error reading Delta Lake table history when the initial transaction logs
+  have been removed. ({issue}`18845`)
+
+## Elasticsearch connector
+
+* Fix query failure when a `LIKE` clause contains multi-byte characters. ({issue}`18966`)
+
+## Hive connector
+
+* Add support for `double` to `varchar` coercion in Hive tables. ({issue}`18930`)
```

Comment:
This is incorrect. It's not about supporting coercions, but allowing certain kinds of table/column evolutions (e.g., changing column type from `double` to `varchar(n)`)